blocks/magazine-summary/magazine-summary.css

```diff
@@ -68,7 +68,7 @@ main .section.magazine-summary-container {
     padding-bottom: 30px;
   }
 
-  .magazine-summary .col1 h4 {
+  .magazine-summary .col1 p {
```

Comment:
Please make sure to keep the font-size, font-family and wont-weight. I did something similar in https://github.com/hlxsites/24life/commit/c7d02fb9bb38c850b788bc9786d65ea0fc5c298b#diff-810eb315e80b427b1942f716d1e23aa42beeafad3208ffea5e249824403a6dbcR50-R55